src/internal_macros.h

```diff
@@ -30,9 +30,13 @@
 #elif defined(_WIN32)
 #define BENCHMARK_OS_WINDOWS 1
 #elif defined(__APPLE__)
-// TODO(ericwf) This doesn't actually check that it is a Mac OSX system. Just
-// that it is an apple system.
-#define BENCHMARK_OS_MACOSX 1
+#include "TargetConditionals.h"
+  #if defined(TARGET_OS_MAC)
+    #define BENCHMARK_OS_MACOSX 1
```

Comment:
Unless you think it makes more sense to check for OSX and iOS separately, but I suppose the benchmark codebase will have a bunch of more complex/longer `ifdef`s then.